ConsoleApp/Program.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+bool Compare (string str1, string str2)
+{
+    if (str1.Length != str2.Length)
+    {
+        return false;
+    } 
+    for (var item = 0; item < str1.Length; item++)
+    {
+        if (str1[item] != str2[item])
+        {
+            return false;
+        }
+        
+    }
+    return true;
+}
+(int alphabeticChars, int digits, int specChar) Analyze (string str)
+{
+    int charCount = 0;
+    int didgitCount = 0;
+    int specCount = 0;
+    for (var item = 0; item < str.Length; item++)
+        if (char.IsLetter(str[item]))
+        {
+            charCount++;
+        }
+        else if (char.IsNumber(str[item]))
+        {
+            didgitCount++;
+        }
+        else
+        {
+            specCount++;
+        }
+    return (charCount, didgitCount, specCount);
+}
+string Sort (string str)
+{
+    str = str.ToLower();
+    var arr = str.ToCharArray();
+    Array.Sort(arr);
+    var str1 = string.Empty;
```

Comment:
here you can simplify
```suggestion
    return new string(arr);
```
string can be initialized with char array